src/app/store/integration-support.service.ts

```diff
@@ -21,7 +23,11 @@ export class IntegrationSupportService {
     this.filterService = restangular.service('integrations');
     this.metadataService = restangular.service('connections');
     this.configService = config;
-    this.mapperBaseUrl = config.getSettings().apiBase + config.getSettings().mapperEndpoint;
+    const apiBase = config.getSettings().apiBase;
```

Comment:
I'm not sure `apiBase` is the right setting.  I thought the real setting for the rest API is `apiEndpoint`, at least a quick check on staging yeilds:

```
{
  "apiEndpoint": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/api/v1",
  "title": "Syndesis",
  "datamapper": {
    "baseJavaInspectionServiceUrl": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/v2/atlas/java/",
    "baseXMLInspectionServiceUrl": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/v2/atlas/xml/",
    "baseJSONInspectionServiceUrl": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/v2/atlas/json/",
    "baseMappingServiceUrl": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/v2/atlas/"
  },
  "oauth": {
    "clientId": "syndesis-ui",
    "scopes": ["openid"],
    "oidc": true,
    "hybrid": true,
    "issuer": "https://syndesis-staging.b6ff.rh-idev.openshiftapps.com/auth/realms/syndesis",
    "auto-link-github": true
  },
  "branding": {
    "logoWhiteBg": "assets/images/syndesis-logo-svg-white.svg",
    "logoDarkBg": "assets/images/syndesis-logo-svg-white.svg",
    "iconWhiteBg": "assets/images/glasses_logo_square.png",
    "iconDarkBg": "assets/images/glasses_logo_square.png",
    "appName": "Syndesis",
    "favicon32": "/favicon-32x32.png",
    "favicon16": "/favicon-16x16.png",
    "touchIcon": "/apple-touch-icon.png"
 }
}
```

The `apiBase` attribute was added when the local dev proxy was added to support going through oauth pages for twitter/salesforce for development instances.  So I'd either suggest we build this URL off of `apiEndpoint` or just add a whole new `mapperEndpoint` setting that contains the full URL of where the client should go to and then we get rid of this other URL parsing :-)

Reply:
I mean, my point is the `config.json` that's part of the openshift templates doesn't contain `apiBase` -> https://github.com/syndesisio/syndesis-openshift-templates/blob/master/generator/03-syndesis-ui.yml.mustache#L106

If we add `mappingEndpoint` then it needs to be added to this file too.